xml_converter/doc/alpha/alpha.md

```diff
@@ -1,13 +0,0 @@
----
-name: Alpha
```

Comment:
We got rid of alpha here, but it was not added anywhere else. Doesn't this make a lot of noise when parsing XML files because some have an alpha field? Maybe just a TODO for #129 should be added here unless there is a better solution I am missing.

Reply:
It is not the best option but since this data is not being stored in the proto file in any way at the moments, there was no simple way to have the generated code ignore it.